### PR TITLE
GitHub issue templates

### PR DESCRIPTION
**Pull request by benwoo1110**
I have added 3 issue templates: help, report a bug and feature request. This will hopefully allow Github issues to be more informative. This is my first PR xD, do give feedback on the template to see what we can add to improve it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+name: Report a bug
+about: Report an Multiverse-Core bug. Only use this if you're 100% sure it's something wrong with Multiverse-Core - otherwise, try "Help!".
```
Comment (nicegamer7) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:3]: Maybe capitalize "bug".

File: .github/ISSUE_TEMPLATE/request-a-feature.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+name: Request a feature
```
Comment (nicegamer7) [inline .github/ISSUE_TEMPLATE/request-a-feature.md:2]: Maybe capitalize "feature".

File: .github/ISSUE_TEMPLATE/report-a-bug.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+name: Report a bug
```
Comment (nicegamer7) [inline .github/ISSUE_TEMPLATE/report-a-bug.md:2]: Maybe capitalize "bug".

**Review by nicegamer7 — DISMISSED** [review]
Looks good to me! Good work.

**Comment by benwoo1110** [comment]
Alright thanks! I will make similar issue templates for the other MV sub modules when I have time (Just copy and change the name from mv-core to mv-<modulename> and some links).

**Pull request merged**